sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5664,6 +5664,20 @@ pi_result piextEventCreateWithNativeHandle(pi_native_handle NativeHandle,
   // need to create a host-visible proxy for it.
   (*Event)->HostVisibleEvent = *Event;
 
+  // Regular events managed by SYCL RT are created with RefCnt == 2 so that
+  // their lifetime is extended beyond that of of the associated pi_event as we
```

Comment:
> beyond that of of the associated pi_event

The lifetime of L0 event and PI event (`pi_event`) are equal for them. We don't destroy PI event while L0 event is still running, and we do destroy both when event is completed and released.

Reply:
Can you suggest the change in wording that would be accurate here?